### PR TITLE
feat: added chips component

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added chips:

>Inline-Chips
Stacked Chips

>Gray Chips
Coloured Chips

chips

### CODE REVIEW

File: components/chips.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+.chip {
+    align-items: center;
+    background-color: var(--primary-100);
+    border-radius: var(--size-28);
+    box-shadow: 0 0 2px var(--primary-300);
+    display: flex;
+    gap: 5px;
+    justify-content: space-between;
+    padding: 8px 8px;
```
Comment (Ronak-Mutha) [inline components/chips.css]: I'll suggest to use only one value if you're giving same values. Also you can use a variable here.

Comment (GuptaEesh) [reply components/chips.css]: Yeah didn't realize that. Maybe amidst tweaking for different values I might have settled for the same unknowingly. 
Done✅

**Review by ZuberDunge — APPROVED** [review]
Looks great. Also, the naming of variables is good. Good to go.

**Comment by GuptaEesh** [comment]
Got green flags from reviewers. Merging the branch now.

**Pull request merged**